src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -219,10 +219,11 @@ private ProgressStepDefinition[] CreateWorkflowSteps(IProgressController control
                 if (!this.host.SonarQubeService.TryGetExportProfile(this.connectionInformation, qualityProfileInfo, language, cancellationToken, out export))
                 {
                     failed = true;
+                    InformAboutQualityProfileToDownload(qualityProfileInfo.Name, qualityProfileInfo.Key, language.Name, true);
                     break;
                 }
-
-                this.NuGetPackages.AddRange(export.Deployment.NuGetPackages);
+                this.NuGetPackages.Add(language, export.Deployment.NuGetPackages);
+                // TODO: add message about how many nuget packages for language
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)